crates/ef-testing/src/evm_sequencer/sequencer/mod.rs

```diff
@@ -29,13 +41,34 @@ impl KakarotSequencer {
                 State::default()
             }
         };
-        let sequencer = Sequencer::new(BLOCK_CONTEXT.clone(), initial_state);
+
+        let block_context = BlockContext {
+            chain_id: ChainId(String::from_utf8(CHAIN_ID.to_be_bytes().to_vec()).unwrap()),
+            block_number: BlockNumber(block_number),
+            block_timestamp: BlockTimestamp(block_timestamp),
+            sequencer_address: compute_starknet_address(&coinbase_address)
```

Comment:
how can we make sure this address is deployed?

since in SSJ, Kakarot attempts to resolve syscall -> get_sequencer_address -> get_evm_address(sequencer_address) -> return coinbase address in eth format

Reply:
The coinbase is deployed during the setup